Automattic/studio
# Studio: Stop server that was previously started before the pull operation

## Related issues

- Fixes https://github.com/Automattic/dotcom-forge/issues/9835

## Proposed Changes

This PR ensures that the server is stopped for a started site before the pull operation starts. Previously, the server was still running for the first part of the pulling process and stopped only when the import event started. 

## Testing Instructions



* Pull the changes from this branch
* Start Studio with `STUDIO_SITE_SYNC=true npm start`
* Click on the `Sync` tab 
* Connect a WP.com site to Studio
* Start the Studio site and confirm it is running
* Start the pull operation on a site
* Observe that the server is stopped immediately and that you can not open `WP-Admin` or `Open site` links or click on a theme thumbnail to open a site

## Pre-merge Checklist
